regression/systemc/Array4/test.desc

```diff
@@ -1,8 +1,10 @@
-CORE
+KNOWNBUG
```

Comment:
Is this a test for an array of empty structs?  If not, maybe we could have one just for sanity checking.

Reply:
No, it was an unintended side-effect that required further debugging and eventually resulted in #3184 and the 5 or so PRs leading up to it. This change thus won't make it into the final version of this PR, once fully rebased.

Reply:
Cool.